velox/functions/prestosql/Probability.h

```diff
@@ -71,9 +71,10 @@ template <typename T>
 struct BinomialCDFFunction {
   VELOX_DEFINE_FUNCTION_TYPES(T);
 
+  template <typename TValue>
   FOLLY_ALWAYS_INLINE void
-  call(double& result, int64_t numOfTrials, double successProb, int64_t value) {
-    static constexpr int64_t kInf = std::numeric_limits<int64_t>::max();
+  call(double& result, TValue numOfTrials, double successProb, TValue value) {
```

Comment:
Please can you double check that numTrials and value are the same type.

Did you try e2e Presto test with this new signatures and check it works ?

Reply:
@aditi-pandit I double checked with the presto Java functions. 

```
    public static double binomialCdf(
            @SqlType(StandardTypes.INTEGER) long numberOfTrials,
            @SqlType(StandardTypes.DOUBLE) double successProbability,
            @SqlType(StandardTypes.INTEGER) long value)
```
It is the same.

I also tried the e2e tests for the new signatures. It works.